sway-ir/tests/dce/dce1.cfg

```diff
@@ -2,14 +2,14 @@
 
 script {
     fn main() -> bool {
-        entry:
+        entry():
```

Comment:
Why is this file `dce1.cfg`, instead of `dce1.ir`?

Reply:
> Why is this file `dce1.cfg`, instead of `dce1.ir`?

Fixed this now.